ui/components/app/incoming-trasaction-toggle/NetworkToggle.js

```diff
@@ -0,0 +1,116 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { Box, Text } from '../../component-library';
+import {
+  AlignItems,
+  BackgroundColor,
+  BlockSize,
+  Color,
+  Display,
+  FlexDirection,
+  JustifyContent,
+  TextColor,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+
+import Tooltip from '../../ui/tooltip';
+import {
+  CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP,
+  CHAIN_IDS,
+} from '../../../../shared/constants/network';
+import ToggleButton from '../../ui/toggle-button';
+import { I18nContext } from '../../../contexts/i18n';
+import Identicon from '../../ui/identicon/identicon.container';
+import UrlIcon from '../../ui/url-icon';
```

Comment:
If these components are intended to represent networks could we use the `AvatarNetwork` component from the component library. This should be the component we are using moving forward for all network identifiers.
https://metamask.github.io/metamask-storybook/?path=/docs/components-componentlibrary-avatarnetwork--docs

Reply:
Thanks George, this is a great suggestion. I didn't know about this component and it is very smooth to use. Now just adapted to `AvatarNetwork` 👍🏻